### DISCUSSION TITLE
This app doesn't sync any prompts

### DISCUSSION THREAD

**Discussion by conderus** [lencx/ChatGPT#1289] [Q&A]
Hi, I was using this app before with promts and now there is something wrong with it. Command / to execute promts stopped working. I uninstalled it and installed again and clicked “Syns Prompts” but there is no chat.prompt.cmd.json file generated again. Prompts are not available anymore in this app?

**Comment by GylanSalih** [comment]
Command / to execute prompts stopped working.
    Uninstalled and reinstalled the app.
    Clicked "Sync Prompts" but no chat.prompt.cmd.json file generated.
    Prompts are not available anymore in the app.
